src/Microsoft.AspNetCore.SignalR.Core/IHubClientsExtensions.cs

```diff
@@ -5,102 +5,113 @@
 
 namespace Microsoft.AspNetCore.SignalR
 {
+    /// <summary>
+    /// Extensions methods for <see cref="IHubClients{T}"/>.
+    /// </summary>
     public static class IHubClientsExtensions
     {
         /// <summary>
+        /// Gets a caller to all hub clients excluding the specified client connections.
         /// </summary>
-        /// <param name="hubClients"></param>
+        /// <param name="hubClients">The clients caller abstraction for a hub.</param>
         /// <param name="excludedConnectionId1">The first connection to exclude.</param>
-        /// <returns></returns>
+        /// <returns>A client caller.</returns>
```

Comment:
`A <typeparamref name="T" /> representing the methods that can be invoked on the clients`

(repeat ad infinitum)